boards/native/Makefile.include

```diff
@@ -4,6 +4,9 @@ NATIVEINCLUDES += -I$(RIOTBASE)/core/lib/include/
 NATIVEINCLUDES += -I$(RIOTBASE)/core/include/
 NATIVEINCLUDES += -I$(RIOTBASE)/drivers/include/
 
+# Set "NATIVE_64BIT=1" to compile for x86_64
+NATIVE_64BIT ?= 0
```

Comment:
Probably yes, but there are a lot of additional changes needed outside the boards folder to make it work.
For example, the codebase has a lot of checks like `ifeq (native, $(BOARD))` that would need to be adapted.
I have a WIP branch that is not yet finished, but it shows the extent of the changes needed [WIP: native64 board](https://github.com/RIOT-OS/RIOT/commit/7f25edfad421c424f1f4b00f97bca1e44a735b4e). 
(This one moves most of the native board into a common folder instead of the way it is handled with `openlabs-kw41z-mini-256kib`).